core/src/validator.rs

```diff
@@ -1257,6 +1276,24 @@ impl Validator {
             repair_quic_endpoint_sender,
         )?;
 
+        if in_wen_restart {
+            info!("Waiting for wen_restart phase one to finish");
+            match wait_for_wen_restart(
+                &config.wen_restart_proto_path.clone().unwrap(),
+                last_vote,
+                blockstore.clone(),
+                cluster_info.clone(),
+            ) {
+                Ok(()) => {
+                    return Err(
+                        "wen_restart phase one completed, will restart to wait for supermajority"
+                            .to_string(),
+                    );
+                }
```

Comment:
I think this should just read "wen_restart phase one completed", and then maybe leave the "restart" logging to whatever is handling/consuming this error

Reply:
Changed.